### PR TITLE
Issue #12 : Added routes and calendar screen

### PR DESCRIPTION
**Pull request by arsh331**
This commit addresses the issue of Calendar page creation.
1. Created routes for different pages of the website.
2. Created the calendar page with a weekly schedule.

### CODE REVIEW

File: apps/frontend/src/Screens/MainScreen.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ const MainScreen = () => {
   }, []);
   return(
   <>
-  <Navbar/>
+  {/*<Navbar/>*/}
```
Comment (maingockien01) [inline apps/frontend/src/Screens/MainScreen.tsx:58]: Probably we can just remove this line.

**Review by maingockien01 — APPROVED** [review]
I have one comment but it's not really a big deal. You can just make one small change.
The rest looks good for me.

File: apps/frontend/src/Screens/MainScreen.tsx (hunk 1 above)
Comment (arsh331) [reply apps/frontend/src/Screens/MainScreen.tsx]: Sorry, I meant to delete that. It's fixed now.

**Review by ntt2402vn — APPROVED** [review]
seems good

**Comment by maingockien01** [comment]
@arsh331 I just added some code to fix the lint step in CI. Feel free to merge whenever you want

**Pull request merged**